scripts/flashing/dlink_recovery_upload.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: MIT
+#
+#   D-Link COVR-P2500 flash utility
```

Comment:
Please explain what this utility is doing.
It looks like you try to not send more than one TCP packet at a time. 
Is this not possible by manipulating the TCP window size?

Reply:
Added explanation: "Upload firmware to the device with waiting for ACK after each TCP packet."
I did not manage to get the upload to work by manipulating the TCP window size or other socket settings.